.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -105,4 +112,4 @@ body:
       description: Please ensure you've completed the following, if applicable.
       options:
         - label: I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).
-          required: false
+          required: false
```

Comment:
should you add a newline?

Reply:
i'm sorry i fail to see why i should. pls explain

Reply:
Ensure new line is present otherwise got some inconsistent diff when different dev edit these files

See `insert_final_newline` on https://editorconfig.org